run-unit-tests.sh

```diff
@@ -38,6 +38,11 @@ sleep 15
 $CMAKE_BUILD_DIRECTORY/tests/Oauth2Test
 docker compose -f tests/oauth2/docker-compose.yml down
 
+# Run BrokerMetadata tests
+docker compose -f tests/brokermetadata/docker-compose.yml up -d
+$CMAKE_BUILD_DIRECTORY/tests/BrokerMetadataTest
```

Comment:
We might need to sleep for a while or find a way to wait until the standalone is ready.

Reply:
@BewareMyPower Thanks, Fixed